src/Service.Tests/SqlTests/RestApiTests/Patch/PatchApiTestBase.cs

```diff
@@ -589,6 +589,52 @@ public virtual async Task PatchOneWithNonNullableFieldAsNull()
             );
         }
 
+        /// <summary>
+        /// Verifies that we throw an exception when an extraneous field provided in the request body for a PATCH operation
```

Comment:
It would be helpful to note in the comment of (insert/patch/put) tests that this validates the behavior of `request-body-strict` when it is
1. Included in runtime config (and set to true)
1. Excluded from runtime config (defaults to true)

Reply:
added a note.